src/log.action.ts

```diff
@@ -42,16 +42,12 @@ export const rememberInitialRecord: Before = async (
   return request;
 };
 
-const getRecordTitle = modifiedRecord => {
-  switch (typeof modifiedRecord.title) {
-    case 'string':
-      return modifiedRecord.title;
-    case 'function':
-      return modifiedRecord.title();
-  }
-  return (
-    Object.values(modifiedRecord).find(v => typeof v === 'string') ?? 'No title'
-  );
+const getRecordTitle = (modifiedRecord, currentAdmin) => {
+  const recordJson =
+    typeof modifiedRecord.toJSON === 'function'
+      ? modifiedRecord.toJSON(currentAdmin)
+      : modifiedRecord;
+  return recordJson.title;
```

Comment:
Let's do `recordJson?.title ?? 'No title'` in case the title is missing for some reason actually